HoloLensCommander/HoloLensCommander/Dialogs/MobileCenterAppsDialog.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<ContentDialog
+    x:Class="HoloLensCommander.MobileCenterAppsDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:HoloLensCommander"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Mobile Center Apps"
+    PrimaryButtonText="Close"
+    PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
+    MaxHeight="800">
+
+    <StackPanel>
+        <Canvas 
+            Width="450" Height="40">
+            <TextBlock 
+                x:Name="UserDisplayNameLabel"
+                Text="Display Name"
+                FontSize="16" 
+                Canvas.Top="10"/>
+            <TextBlock 
+                x:Name="UserDisplayName"
+                Text="{Binding Path=DeviceAddress}"
+                ToolTipService.ToolTip="The address of the target device"
+                FontSize="16"
+                Width="347"
+                Canvas.Left="103" Canvas.Top="10"/>
+        </Canvas>
+        <Canvas
+            Width="450" Height="440">
+            <TextBlock 
+                x:Name="UserEmailLabel"
+                Text="Email"
+                FontSize="16"/>
+            <ListBox 
```

Comment:
Do we want to make this multi-select?

Reply:
@davidkline-ms Yes, we could make it multi-select to allow for multiple downloads at once. That would require some additional steps for making the download call async and queuing up concurrent downloads, correct?

Reply:
You are probably right. Unless the AppCenter has an endpoint for "install these". Probably something best left for the next rev :)

Reply:
Do my knowledge there is not. I am using the download URL and then caching that downloaded data. There is "install" api other than to install to a mobile device for testing. 
![imgo](https://user-images.githubusercontent.com/14004320/35407824-f0abb4f2-01c1-11e8-9bbf-5e270b170f20.jpg)